assets/src/apps/authoring/components/SequenceEditor/SequenceEditor.tsx

```diff
@@ -391,7 +410,8 @@ const SequenceEditor: React.FC<any> = (props) => {
                 }}
               >
                 <i className="fas fa-clipboard align-text-top mr-2" /> Copy{' '}
-                {item.custom.isLayer ? 'Layer' : 'Screen'} ID
+                {item.custom.isLayer ? 'Layer ' : item.custom.isBank ? 'Qustion Bank ' : 'Screen '}{' '}
```

Comment:
same here with the nested tertiary. also typo in "Question" ... maybe a good idea to put the strings "Layer", "Question Bank" and "Screen" into constants since they are reused a lot in this file that way can prevent typo and also allow for quick renaming if we decide to call it something other than "Question Bank"